Add reading section (#13)
open-education-hub/methodology: description.md

### Reading

The reading content type is the most common form of storing educational material (or information, in general).
As the name suggests the content is developed to be read, therefore it comes in the form of text.

To develop reading content we use the [Markdown syntax](https://www.markdownguide.org/basic-syntax) and the following guidelines:

- the heading level of a title is given by the number of `#`s that precede it.
For example, the title of a document is preceded by a single `#`, whereas the chapters are preceded by two.
- each sentence is written on a new line.
- paragraphs are separated by an empty line.
- links should be used as hyperlinks by using the `[]()` syntax as in [this example](https://theuselessweb.com/).

> 

Reviewer: There are a few more rules enforced by our linters such as leaving blank lines before and after lists, using the same number for every item of a numbered list, such as:

```
1. first item

1. second item
```

But listing them exhaustively would be pointless. I'm not sure which ones are relevant to this section. Contributors will "discover" them anyway when they want to raise a PR, but not knowing about them beforehand will surely create frustration.

A good thing you can do is to give a link to [super-linter](https://github.com/super-linter/super-linter#readme) so people can at least know what behemoth lies ahead of them.

Author: @teodutu I added a couple of sentences.

The above are enforced by automated linter checks.

When developing such materials we recommend that:

> - the reading material is succint and concise, following the "les is more" principle.
> - the first sentence of each section describes what is the intended purpose of the section.
> - if the reading material can be expressed in a more visual manner, the latter is preffered.

Reviewer: ```suggestion
- the reading material be concise, following the "less is more" principle.
- the first sentence of each section describe the intended purpose of the section.
- if the reading material can be expressed in a more visual manner, the latter is preferred.
```

- succinct (written with an extra `c`, despite this spelling being incorrect in other languages, such as Romanian) == concise
- use the [subjunctive mood](https://www.scribbr.com/verbs/subjunctive-mood/). It's indeed a relic, but so are crocodiles.

### Reading

The reading content type is the most common form of storing educational material (or information, in general).
As the name suggests the content is developed to be read, therefore it comes in the form of text.

To develop reading content we use the [Markdown syntax](https://www.markdownguide.org/basic-syntax) and the following guidelines:

- the heading level of a title is given by the number of `#`s that precede it.
For example, the title of a document is preceded by a single `#`, whereas the chapters are preceded by two.
- each sentence is written on a new line.
- paragraphs are separated by an empty line.
- links should be used as hyperlinks by using the `[]()` syntax as in [this example](https://theuselessweb.com/).

To enforce the guidelines we use [automated linter checks](https://github.com/super-linter/super-linter#readme).
Besides the mentioned guidelines, the linter implements checks for other minor style conventions (such as how to define lists, where should new lines be placed and others).
However, for the majority of cases, the above should be sufficient.

> When developing such materials we recommend that:

Reviewer 2: This should come at the beginning, after the definition of the reading content type. The reader should be aware of general guidelines and principles before giving concrete action points.
